### DISCUSSION TITLE
What is the best place for the remove-cell button?

### DISCUSSION THREAD

**Discussion by tskarhed** [fonsp/Pluto.jl#773] [Pluto development]
Having the x inside the input box makes unclear that it removes the cell. It feels like it removes the content of the input box.

My suggestion would to be to put it next to the entire cell - like this:
![image](https://user-images.githubusercontent.com/1438972/102000453-e14d8d00-3ce7-11eb-8926-71ddb0abd7f8.png)

Then you know that the action is on the cell, rather than the content of the input box.

Share any additional thoughts that you might have on this!

**Comment by fonsp** [comment]
Can you be clear about which problem you want to solve?

> It feels like it removes the content of the input box.

That's close to what it does, but the problem is...

**Reply by tskarhed** [reply]
It is mostly a question of user experience. It's position makes it unclear what the action of the button is. A common pattern when placing a cross inside the input is that it just clears the input from all text - not that it removes the entire element and a bit more.

My suggestion would be to place it outside of the box itself so you know that you remove the entire cell (which includes the input box and the output content)

**Comment by fonsp** [comment]
_(Normally I would close this issue and submit it to our suggestion box instead (https://github.com/fonsp/Pluto.jl/issues/182), but I have converted it to a Discussion under the "Development" topic as an experiment)_

**Reply by tskarhed** [reply]
Ah, sorry, I didn't see that! I didn't find any guidelines regarding creating issues in CONTRIBUTING.md. I'll create a pull request with a link to that issue!

**Reply by fonsp** [reply]
Awesome! Put it at the top
